docs/usage/faq.md

```diff
@@ -39,6 +39,26 @@ The Renovate team will only create bugfixes for an older version if the hosted a
 If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
 If you're self hosting Renovate, use the latest release if possible.
 
+## Renovate core features not supported on all platforms
```

Comment:
The idea here is to list the "core" features that we have on some, but not all platforms. Think of things like the Dependency Dashboard, automerge support, hosted app, automerge support, etc.